### ISSUE TITLE
Add sideeffects: false to package.json

### ISSUE THREAD

**Issue by travigd** [browserify/crypto-browserify#197]
See title.

Allows for deadcode elimination when not the module is imported but not actually used.

This comes up for me when using NextJS where one frequently writes server side code as a `getServerSideProps` function. I want to import Node's crypto module but don't need it for the browser -- but webpack is unable to remove the import even though Next does remove the `getServerSideProps` function).

The workaround is to add

```js
module.exports = {
  node: {
    crypto: "empty"
  }
}
```

to the webpack config.

**Comment by mitchheddles** [comment]
@travigd Can you use a dynamic import instead?
```ts
export async function getServerSideProps() {
  const crypto = await import('crypto');
}
```

**Comment by travigd** [comment]
Yes, but this should still be fixed. :)

I'm currently just using the webpack workaround described above.

**Comment by coltongit** [comment]
Could probably integrate this into PR #200 tbh. Will here shortly

**Comment by ljharb** [comment]
Probably best not to conflate multiple things in the same PR.

**Comment by coltongit** [comment]
Would it be best to rather make a separate PR after #200 is pushed to have this fixed?

**Comment by ljharb** [comment]
I'd think so.

**Comment by coltongit** [comment]
PR #201 will fix this issue

**Comment by ljharb** [comment]
Fixed in b638139cc86e560ce65533c00e5f72b95926ff59

**Comment by rawr51919** [comment]
> Fixed in [b638139](https://github.com/browserify/crypto-browserify/commit/b638139cc86e560ce65533c00e5f72b95926ff59)

glad you got around to it thanks :)
